src/shared/components/palette/organisation/write-organisation-field.component.ts

```diff
@@ -90,10 +90,28 @@ export class WriteOrganisationFieldComponent extends AbstractFieldWriteComponent
     if (organisation.postCode && organisation.postCode.toLowerCase().includes(this.trimAll(lowerOrgSearchText))) {
       return true;
     }
-    // noinspection RedundantIfStatementJS
     if (organisation.name && organisation.name.toLowerCase().includes(lowerOrgSearchText)) {
       return true;
     }
+    if (organisation.addressLine1 && organisation.addressLine1.toLowerCase().includes(lowerOrgSearchText)) {
```

Comment:
```suggestion
    return organisation.addressLine1 && organisation.addressLine1.toLowerCase().includes(lowerOrgSearchText);
```
You could simply do a return of the conditional statement.

Reply:
for the last if statement yes, but it can't return from this addressLine1 statement as there are other ifs statment need to be judged.
to keep the code to be more readable I would leave this.